### ISSUE TITLE
review of beta version: 01.00.29b1

### ISSUE THREAD

**Issue by taylor13** [cmip6dr/CMIP6_DataRequest_VariableDefinitions#368]
Would it be possible to address issue 351 and 353 for this release?

variable toz in AERday:  long_name changed from "Total Column Ozone" to "Total Ozone Column".  I think the old name is much more commonly used.

variable edt in CFsubhr: the new long_name is "Eddy Diffusivity Coefficients for Temperature".  Shouldn't "Coefficients" be "Coefficient"?

variable vegHeight: the new long_name is "Height of the vegetation canopy." (lower case and ending with a period), but it probably should be "Height of the Vegetation Canopy"  

variable fHarvestToProduct in Emon: title case for the new long_name would read: "Harvested Biomass That Goes Straight into Atmosphere"

variable fNAnthDisturb in Emon: title case for the new long_name would read: "Nitrogen Mass Flux out of Land Due to Any Human Activity"

variable fProductDecompLut in Emon: title case for the new long_name would read: "Flux from Wood and Agricultural Product Pools on Land Use Tile into Atmosphere"

variables lwtoaasdust, lwtoacsaer, and lwtoacsdust in Emon:  I would place "TOA" at the beginning of the long_names, rather than at the end since it modifies radiative forcing, not the constituent (e.g., it isn't "Dust at the TOA")

variables ocontemppadvect, ocontemppmdiff, and ocontemppsmadvect:  the long_names end with a phrase like "due to parameterized mesoscale diffusion".  Title case says they should end with "Due to Parameterized Mesoscale Diffusion"

check capitalization of long_names for: opottemppmdiff, opottemppsmadvect, opottemprmadvect, opottemptend, osaltdiff, osaltpadvect, osaltpmdiff, osaltrmadvect, osalttend,

variables "t2" :  I think the new long_name "Air Temperature Squared" shoujld read "Mean-Square Air Temperature"  or something like that to distinguish it from the mean temperature squared.   Similarly for variables "u2", "v2" and "wap2".

variable "wtd":  long_name capitalization.

variable jo2 in EmonZ:  The new long_name says "Photolysis Rate of Molecular Oxygen", but wouldn't that include photolysis rate of O3 in addition to O2?  I think only O2 is included in jo2.

treeFrac:  For consistency with other names, I think the long_name should be "Tree Cover Percentage".

limnsw:  long_name should be in title case.

snc:  long_name should be "Snow Area Percentage"

chlos longname should be title case.

masscello:  I think the long_name should read:  "Ocean Grid-Cell Mass per Unit Area"

check title case for long_names:  bfeos, bsi, bsios, cfc12, chl, chlcalc, chlcalcos, etc.  In particular "expressed" should be capitalized.

sidconcdyn and sidconcth:  These should be "Percentage Change Rate", not "Percentage Change".  (or "tendency"?)

check title case for long_names:  siitdconc, siitdsnconc, siitdsnthick, sisnconc, 

check title case for sftlf.

**Comment by martinjuckes** [comment]
jo2: the definition of the standard name (`photolysis_rate_of_molecular_oxygen`) says that "The chemical formula for molecular oxygen is O2", so I think this is OK. This does appear to reflect the common usage "molecular oxygen" means just the O2, not all oxygen in molecular form -- I can see why you might expect a different meaning.

sidconcdyn and sidconcth: ".. Percentage Change from ..." --> " .. Percentage Tendency Due to ...";

I think the units should also change from "s-1" to "% s-1". 

also: "Rate of C accumulation" --> "Rate of Carbon ....";
"Total N ..." --> "Total Nitrogen ...>";
"(aka frazil)" --> "(frazil)";
"Eater" --> "Water";
"sw-rf" --> "Shortwave flux due to"
"CFC-11", "CFC-12" --> "CFC11", "CFC12" for consistency
"above ground", "below ground" --> "Above-ground", "Below-ground"  (hyphenated because it is used as an adjective);
od550aerso, od550so4so: long name shortened;

There are some further issues with capitalization --- see next post.

**Comment by martinjuckes** [comment]
Most words should be capitalised in long_names .. in setting some rules I'm trying to follow https://english.stackexchange.com/questions/14/which-words-in-a-title-should-be-capitalized .. which is consistent with other credible sources.

The following should then be left in lower case:
2 ::  as at by in nm of on or to
3 ::  and any but for non nor not out per the
4 ::  into onto over than with

There are a number of "words" which have special capitalisation:
2 ::  10 1H 1m 2H 2m 3D 50 C3 C4 CO D: H2 N2 NO O2 O3 OD OH Ox XY pH
3 ::  10m 13C 14C 17O 18O 20C CH4 CO2 CWD DMS HCl HO2 N2O NH3 NH4 NHx NO2 NO3 NOx NOy NPP O1d PAN PBL PO2 SF6 SO2 SO4 TEM TOA hPa
4 ::  1000 100m 300m 700m C2H2 C2H6 C3H6 C3H8 CFAD MISR PCO2 PM10
5 ::  10hPa 13CO2 14CO2 2000m 4XCO2 550nm CFC11 CFC12 DI14C ISCCP MODIS NMVOC PM1.0 PM2.5 UGRID
6 ::  CFC113 HCFC22
7 ::  CALIPSO PARASOL

Other changes made:
SWE --> "snow water equivalent"
"20C  isotherm depth" -->  "Depth of 20 Degree Celsius Isotherm"

Applying above rules (putting all the first set in lower case, except for the first word), capitalizing all other words except the list of special words, gives a number of changes. Here is a random selection of 25:
```
----------------
Aerosol backscatter coefficient
Aerosol Backscatter Coefficient
----------------
CALIPSO Cloud Cover Percentage Undetected by CloudSat (as percentage of area covered)
CALIPSO Cloud Cover Percentage Undetected by Cloudsat (as Percentage of Area Covered)
----------------
Carbon Mass in Below-Ground Litter
Carbon Mass in Below-ground Litter
----------------
Change in soil moisture
Change in Soil Moisture
----------------
CloudSat Radar Reflectivity CFAD
Cloudsat Radar Reflectivity CFAD
----------------
Direct surface albedo for each band
Direct Surface Albedo for Each Band
----------------
Formaldehyde volume mixing ratio
Formaldehyde Volume Mixing Ratio
----------------
liquid water path
Liquid Water Path
----------------
MODIS Optical Thickness-Particle Size Joint Distribution, Liquid
MODIS Optical Thickness-particle Size Joint Distribution, Liquid
----------------
Mass Concentration of Calcareous Phytoplankton expressed as Chlorophyll in Sea Water
Mass Concentration of Calcareous Phytoplankton Expressed as Chlorophyll in Sea Water
----------------
Mass of 13C in all terrestrial carbon pools
Mass of 13C in All Terrestrial Carbon Pools
----------------
Net conductive heat fluxes in ice at the bottom
Net Conductive Heat Fluxes in Ice at the Bottom
----------------
Net latent heat flux over sea ice
Net Latent Heat Flux over Sea Ice
----------------
net primary production on grass tiles
Net Primary Production on Grass Tiles
----------------
Ocean Kinetic Energy Dissipation Per Unit Area due to XY Friction
Ocean Kinetic Energy Dissipation per Unit Area Due to XY Friction
----------------
Surface Upwelling Clear-Sky Shortwave Radiation for each band
Surface Upwelling Clear-sky Shortwave Radiation for Each Band
----------------
TOA Outgoing Clear-Sky Shortwave Radiation for each band
TOA Outgoing Clear-sky Shortwave Radiation for Each Band
----------------
Temperature Tendency due to Non-orographic Gravity Wave Dissipation
Temperature Tendency Due to Non-orographic Gravity Wave Dissipation
----------------
Tendency of Air Temperature due to Numerical Diffusion
Tendency of Air Temperature Due to Numerical Diffusion
----------------
Tendency of eastward wind due to TEM northward advection and Coriolis term
Tendency of Eastward Wind Due to TEM Northward Advection and Coriolis Term
----------------
Tendency of Sea water Salinity Expressed as Salt Content Due to Parameterized Eddy Advection
Tendency of Sea Water Salinity Expressed as Salt Content Due to Parameterized Eddy Advection
----------------
Total Carbon Mass Flux from Vegetation to Litter as a Result of Leaf, Branch, and Root Senescence
Total Carbon Mass Flux From Vegetation to Litter as A Result of Leaf, Branch, and Root Senescence
----------------
total net production of anthropogenic secondary organic aerosol
Total Net Production of Anthropogenic Secondary Organic Aerosol
----------------
X-component of sea-ice mass transport
X-component of Sea-ice Mass Transport

```

**Comment by martinjuckes** [comment]
There are 592 long names that will be adjusted with above capitalization rules.

**Comment by taylor13** [comment]
Fantastic.  I don't think a lot of folks would have bothered, but it sure cleans up stuff.

**Comment by martinjuckes** [comment]
I'm hoping to get the code I'm using into a re-usable state so that it will be less effort in the future, and also write a short style guide -- to get others to do the work up front next time.

**Comment by martinjuckes** [comment]
Concentration of DI14C --> Dissolved Inorganic 14Carbon Concentration (for consistency with 13Carbon case)

**Comment by martinjuckes** [comment]
Terms like "100m" should be "100 m" [ (see [NIST guide for use of SI units](https://physics.nist.gov/cuu/pdf/sp811.pdf), page vi]. This approach is generally followed for "hPa", but "m" is often joined with the preceding number on 01.00.29b1. E.g. `Maximum Wind Speed of Gust at 10m` .. if  `Maximum Wind Speed of Gust at 10 m` looks odd, it may be better to use `Maximum Wind Speed of Gust at 10 Metres`. 

On the other hand, Cambridge Uni. recommends omitting the space [here](https://www.cam.ac.uk/brand-resources/guidelines/editorial-style-guide).

**Comment by taylor13** [comment]
or "10 meters" ???   In any case I'd be happy with any of the options you describe (so I guess you should just go with your preference.

Regarding "molecular oxygen", it does seem that it is taken to mean O2 (by those who use the term).  I wish they had  been a bit more specific and used something like "diatomic molecular oxygen", but as long as those in field won't be confused, I'm fine with the common usage.

**Comment by martinjuckes** [comment]
We can add "diatomic" into the long name for "molecular" oxygen ... I agree that it will make it clearer for our community.

**Comment by martinjuckes** [comment]
I've created a [style guide for long names](https://github.com/cmip6dr/cmip7_forward_look/blob/master/StyleGuideForTitles.md) ... setting out some rules. These are encoded in a python script so that I can apply them consistently. There are some grey areas, but the main thing is consistency, and that is now pretty good. I've cleaned up some messy hyphenation (e.g. replacing `MODIS Optical Thickness-Particle Size joint distribution, liquid` with `MODIS Joint Distribution of Optical Thickness and Particle Size, Ice`). 

For "10 metres", I decide to stay with "10m" in the end, but leave a space when the unit is spelled out (as in "20 day").
